tests/QueryLanguage/Processor/Doctrine/PhpCr/ProcessorTest.php

```diff
@@ -235,6 +237,50 @@ public function testPageSizeOptionShouldWork(array $params): void
         self::assertCount(3, $result);
     }
 
+    public function provideParamsForDefaultOrder(): iterable
+    {
+        yield [false, true, '$order(name)'];
+        yield [false, true, 'name'];
+        yield [false, true, 'name, desc'];
+        yield [true, false, '$eq(name)'];
+    }
+
+    /**
+     * @dataProvider provideParamsForDefaultOrder
+     */
+    public function testShouldOrderByDefaultField(bool $exception, bool $valid, string $defaultOrder): void
```

Comment:
Same as the ORM one: I would prefer to have these tests separated instead a single. 
Something like `testOrderByDefaultFieldShouldThrowOnInvalidOptions` and `testOrderByDefaultFieldShouldWork`

Reply:
Done